### ISSUE TITLE
Suggestion On How To Improve Vitality

### ISSUE THREAD

**Issue by Zeninari** [Jalopy-Mods/Vitality#1]
So i don't want to be rude but ill be blunt, up to you if you consider that rude but i want to give my honest blunt feedback on the mod.

the bars are amazing if it wasn't so hard to maintain, and what i mean is that the bars fill up so quickly that I'm basically' panicking on trying to keep them in check. so i wanted to propose a way in the settings to decrease the rate in which the bar fills up by x% (preferably at least 50% of its rate[though i prefer if its a bit more flexible than just 50% to figure out the sweet spot on what i want it to be] to make my journey more enjoyable) i also have more suggestions in the short list below. the 4th one is more of a cool thing i thought of, up to you if you pay it any mind. for now I'm disabling the mod because i just can't manage the bars they fill up too quickly and its too stressful.

for clarification i went through 6 bottles of water, 7 food items, a bunch of coffee and such during ONE trip between borders which hurts a lot. if anything i want my first suggestion to be added in some way.

so in short:

1.  add a setting that allows us to decrease the % in which the bars fills up, even better if its per bar for even more customization
2. add the ability to see when the bar is at a critical point, (for example the main bar can be blue or green while critical can be red)
3.  the ability to see what number the bar is at (for example if the hunger bar is full it would also say its 100%), this would make the bars more manageable as being able to physically see what its at is very helpful
4. idk if its possible, but the ability to actually use the cigarette by lighting it using the car lighter would be sick, also finally fixes it having no purpose. even cooler if it allows us to actually smoke the cigarette using a custom model x'D

Edit: also idk how to add a label if i can ;-;

**Comment by theLeaxx** [comment]
done, will release soon

**Comment by Zeninari** [comment]
didn't expect this thanks for this! ill check it out later when i get home and have some free time <3
